_src/static/sass/components/_colors.scss

```diff
@@ -232,18 +232,6 @@
         background: palette(accent, x-dark);
     }
 
-    &.mid-sat {
```

Comment:
These are being removed from brand-accent. I believe we still want to have the saturated values for this configured color, no? Remember, a company/org may not have gray as their accent like we do.